scripts/release-notes.txt

```diff
@@ -1,5 +1,14 @@
 Release Notes of the ESBMC model checker
 
+*** Version 6.10
+* Lots of bugs cases relating unions have been fixed
+* Compound Literals are now supported
+*  _ExtInt struct fields now have proper padding and alignment
```

Comment:
Remove one of the two spaces at the beginning of this line.

Reply:
Done